libgfortran/libgfortran.h

```diff
@@ -62,13 +62,19 @@ see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
 # include "quadmath_weak.h"
 #endif
 
-#ifdef __MINGW32__
```

Comment:
The change has been done for a testing purpose and based on CI results it looks like it also works